### PR TITLE
Validate

### PR DESCRIPTION
**Pull request by mnatour2**

### CODE REVIEW

File: Login/index.html [hunk 1]
```diff
@@ -20,47 +20,53 @@
         </nav>
     </header> -->
     <div id="content">
-        <form action="Get">
+        <form action="Get"">
```
Comment (abedmurrar) [inline Login/index.html]: extra quotation here.

**Review by abedmurrar — CHANGES_REQUESTED** [review]
good job, only one note check it

File: Login/index.html (hunk 1 above)
Comment (mnatour2) [reply Login/index.html]: fixed

Comment (MajdTaweel) [reply Login/index.html]: Remove the action attribute altogether while you're at it @mnatour2.

**Review by MajdTaweel — APPROVED** [review]
Good job.

File: Login/index.html (hunk 1 above)
Comment (mnatour2) [reply Login/index.html]: done

**Review by abedmurrar — APPROVED** [review]

**Pull request merged**